nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/SequenceFileWriterImpl.java

```diff
@@ -94,8 +94,10 @@ public void process(InputStream in, OutputStream out) throws IOException {
 
                 try (final FSDataOutputStream fsDataOutputStream = new FSDataOutputStream(bwos, new Statistics(""));
                         final SequenceFile.Writer writer = SequenceFile.createWriter(configuration,
-                                fsDataOutputStream, Text.class, InputStreamWritable.class, compressionType,
-                                new DefaultCodec())) {
+                                SequenceFile.Writer.stream(fsDataOutputStream),
+                                SequenceFile.Writer.keyClass(Text.class),
+                                SequenceFile.Writer.valueClass(InputStreamWritable.class),
+                                SequenceFile.Writer.compression(compressionType, new DefaultCodec()))) {
```

Comment:
Is it expected to use DefaultCodec? I saw we have a property for compression codec, shouldn't we propose something to use this property at this moment?